packages/react-meteor-data/package.json

```diff
@@ -1,11 +1,12 @@
 {
   "name": "react-meteor-data",
   "dependencies": {
+    "@testing-library/react": "^10.0.2",
+    "@types/meteor": "^1.4.66",
+    "@types/react": "^16.9.34",
+    "fast-deep-equal": "^3.1.3",
```

Comment:
Just wondering why `fast-deep-equal` is a dependency?

Reply:
I'm sure just for testing. Most (all?) of those look like they could be devDependencies.

Reply:
That's what I was thinking however I can't see it being referenced anywhere. They should be devDependencies but since this isn't shipped as an npm package it doesn't really matter.

Reply:
Correct, just for testing. Technically, these are all dev dependencies, since they should not be present for release.